pom.xml

```diff
@@ -19,21 +19,6 @@
 		</license>
 	</licenses>
 
-	<developers>
```

Comment:
we hebben een fork gemaakt, maar is het dan netjes om dit allemaal te verwijderen? wanneer we deze change willen terug geven aan de maintainer dan is dit niet handig denk ik.

Reply:
Tja, het is al een fork van een andere repo. We hebben een fork van een fork gemaakt. Die gast heeft dit toegevoegd. Het is ook maar voor een klein deel zijn code